### DISCUSSION TITLE
enable the option to use a existing huggingface cache

### DISCUSSION THREAD

**Discussion by mjkrakowski** [oobabooga/text-generation-webui#1611] [Ideas]
On my win10 i use python native and in a wsl2 ubuntu sub system. so far i am able to use a single HUGGINGFACE_HUB_CACHE for both. i think it would be great to use it for the textgen webui too.

**Comment by sinogermany** [comment]
@mjkrakowski  See https://github.com/oobabooga/text-generation-webui/discussions/3354#discussioncomment-7150378 , enjoy 🤗

**Comment by mjkrakowski** [comment]
many thanks
